pages/index.js

```diff
@@ -1,36 +1,13 @@
 import Recipes from "../components/Recipes/Recipes";
 import SelectionButton from "../components/SelectionButton/SelectionButton";
 import SelectionDetails from "../components/SelectionDetails/SelectionDetails";
-import CookingButton from "../components/CookingButton/CookingButton";
 
-export default function RecipesPage({
-  isSelectionMode,
-  handleToggleSelectionMode,
-  recipesSelection,
-  onToggleSelection,
-  selectedRecipesCount,
-  totalCookingTime,
-}) {
+export default function RecipesPage({}) {
```

Comment:
if you dont need to destructure you can delete {}

Reply:
i will change that in the next story.